src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -203,6 +203,18 @@
 	flex-direction: column;
 }
 
+.tabs-widget-container .yellow {
```

Comment:
We should name these styles after the status types rather than the color since themes may not necessarily display these colors. I'd recommend `.status-warning`, `.status-error`, etc. if we need these css rules.

Reply:
they'll be removed :)